src/python/pants/backend/go/util_rules/assembly.py

```diff
@@ -68,6 +68,14 @@ class FallibleAssembleGoAssemblyFilesResult:
     stderr: str | None = None
 
 
+def _is_runtime_package_import_path(import_path: str) -> bool:
+    """Returns True if the import path is from one of the Go runtime packages in stdlib which needs
+    special handling."""
+    if import_path in ("runtime", "reflect", "syscall", "internal/bytealg"):
```

Comment:
Is this list documented anywhere?

Reply:
Just in the `go` sources unfortunately (as with much of this build-related code). I'll link to the relevant lines in the `go` sources.